cogeo_mosaic_tiler/handlers/app.py

```diff
@@ -94,29 +84,38 @@ def _create(
     minzoom = int(minzoom) if isinstance(minzoom, str) else minzoom
     maxzoom = int(maxzoom) if isinstance(maxzoom, str) else maxzoom
     min_tile_cover = (
-        float(min_tile_cover) if isinstance(min_tile_cover, float) else min_tile_cover
+        float(min_tile_cover) if isinstance(min_tile_cover, str) else min_tile_cover
     )
```

Comment:
For all of these, why not just coerce to the right type?
```py
minzoom = int(minzoom)
maxzoom = int(maxzoom)
min_tile_cover = float(min_tile_cover)
```
is much clearer than having lots of `if isinstances`, and has the same effect.

Reply:
```
In [1]: int(None)                                                                                                                                                                                                                                     
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-1-af2123a46eb2> in <module>
----> 1 int(None)

TypeError: int() argument must be a string, a bytes-like object or a number, not 'NoneType'
```